### ISSUE TITLE
Harmonize security considerations

### ISSUE THREAD

**Issue by danvangeest** [lamps-wg/cms-kyber#4]
Harmonize Security Considerations section between this and draft-ietf-lamps-kyber-certificates.

**Comment by seanturner** [comment]
Agreed. I went for the refer to [draft-sfluhrer-cfrg-ml-kem-security-considerations](https://datatracker.ietf.org/doc/draft-sfluhrer-cfrg-ml-kem-security-considerations/), but that might be overkill. At a minimum, it adds a dependency. Also, not sure how the IESG will react to an informative reference for all of the security considerations.
